charts/consul/templates/client-daemonset.yaml

```diff
@@ -353,6 +353,10 @@ spec:
             {{- toYaml .Values.client.resources | nindent 12 }}
             {{- end }}
           {{- end }}
+          {{- if .Values.client.containerSecurityContext.client }}
+          securityContext:
+            {{- toYaml .Values.client.containerSecurityContext.client | nindent 12 }}
+          {{- end }}
```

Comment:
I notice the Pod `securityContext`s are wrapped in a:

```
{{- if not .Values.global.openshift.enabled }}
securityContext:
    ...
{{- end }}
```

I don't know that much about openshift, would these container Security contexts need to have this same conditional wrapper too?